app/components/containerConsole/containerConsole.html

```diff
@@ -33,8 +33,13 @@
                 </select>
               </div>
             </div>
+            <div class="col-sm-3">
+              <div class="input-group">
+                <input class="form-control" type="text" name="user" ng-model="state.user" placeholder="User">
+              </div>
+            </div>
             <!-- !command-list -->
-            <div class="col-sm-8">
+            <div class="col-sm-5">
```

Comment:
It looks kinda weird with current size of tooltip so we should either change a text or size of a tooltip

<img width="1019" alt="screen shot 2017-07-04 at 16 21 52" src="https://user-images.githubusercontent.com/6943514/27831831-fa1e04ce-60d4-11e7-8023-49cb3bfdfe67.png">